.travis/before_script.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
 set -ev
 
-mozdownload --type daily --branch mozilla-aurora
+if [[ $LOCALE = 'en-US' ]]
+then
+    mozdownload --type daily --branch mozilla-aurora
+else
+    mozdownload --type daily --branch mozilla-aurora --locale ru
```

Comment:
Maybe we can also move the branch name as global env variable so this file would look identical across all branches.